test/support/test_helpers.ex

```diff
@@ -3,6 +3,7 @@ defmodule RemoteRetro.TestHelpers do
   alias RemoteRetro.{Repo, User, Vote}
```

Comment:
To avoid repeating `RemoteRetro`, you can leverage the `alias` macro, seen here.

Reply:
yessss.